### PR TITLE
Capstone scoring script

### PR DESCRIPTION
**Pull request by CatarinaSilva**

### CODE REVIEW

File: scripts/capstone_scoring.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    for ddp in qs:  # loop through each entry in DueDataPoint
+        outcome = bool(json.loads(ddp.datapoint.outcome))
+        data = json.loads(ddp.datapoint.data)
+        if ddp.response_status != 200:  # Missing or bad response
+            predictions.append(not outcome)
+            outcomes.append(outcome)
+
+        else:
+            try:
+                prediction = json.loads(ddp.response_content)['prediction']
```
Comment (buedaswag) [inline scripts/capstone_scoring.py:38]: should this be `prediction = json.loads(ddp.response_content)['outcome']` ?

Comment (buedaswag) [reply scripts/capstone_scoring.py:38]: https://app.swaggerhub.com/apis/l6837/batch4-capstone/1.0.0#/

Comment (buedaswag) [reply scripts/capstone_scoring.py:38]: when scoring the students i used
```python
try:
    content = json.loads(ddp.response_content)
    prediction = content.get('outcome')
except (json.JSONDecodeError, KeyError, AttributeError):
```

**Review by hcastilho — APPROVED** [review]
Merging to close PR

**Pull request merged**